packages/components/src/FilterDateRangePicker/subcomponents/FilterDateRangePickerField/types.ts

```diff
@@ -4,3 +4,43 @@ export type DateRangeFieldValidationMessage = {
   dateStart?: ValidationMessage
   dateEnd?: ValidationMessage
 }
+
+export type DateRangeFieldActions =
```

Comment:
Since the reducer is internal, I don't think it's necessary to be distributing this type. I'd just place it with the reducer instead. Up to you though.